### ISSUE TITLE
Translations not updating inside transition element

### ISSUE THREAD

**Issue by Gerche** [leonardovilarinho/vue-multilanguage#23]
When adding a [vue transition](https://vuejs.org/v2/guide/transitions.html) around a translation and you try to update the language these strings will not be translated (all other strings outside the transition will however). As soon as I remove the transition tags the translations work as intended again.

```
<transition name="someAnimation">
    <div>
        <p v-lang.someValue ></p>
    </div>
</transition>
```

**Comment by leonardovilarinho** [comment]
I am working it.

**Comment by leonardovilarinho** [comment]
Fix 1af6c5d
